Source/WebCore/rendering/AutoTableLayout.cpp

```diff
@@ -171,9 +171,10 @@ void AutoTableLayout::fullRecalc()
             unsigned effCol = m_table->colToEffCol(currentColumn);
             unsigned span = column->span();
             if (!colLogicalWidth.isAuto() && span == 1 && effCol < nEffCols && m_table->spanOfEffCol(effCol) == 1) {
+                m_layoutStruct[effCol].usedZoom = column->style().usedZoomForLength();
```

Comment:
how does this work with column spanning where a cell spans multiple columns that each have different zoom values?